contracts/Cross-chain/OmnichainGovernanceExecutor.sol

```diff
@@ -271,7 +288,7 @@ contract OmnichainGovernanceExecutor is ReentrancyGuard, BaseOmnichainController
         bytes memory srcAddress_,
         uint64 nonce_,
         bytes memory payload_
-    ) internal virtual override whenNotPaused {
+    ) internal virtual override {
```

Comment:
Why don't we keep also here the `whenNotPaused`? The original intention was not to accept messages if the contract is paused

Reply:
Yes, the original idea was to temporarily block the pathway if the contract was `paused`, however I later discovered an attack vector described below.

Suppose the bridge is hijacked and a bad actor sends a malicious proposal, it comes to our knowledge and we paused the bridge immediately. While investigating, what if the bad actor bombards thousands of proposals? At this time, all the malicious proposals will be in blocked state, and if we resume the bridge, all of these malicious proposals will be queued automatically because layerzero retries the blocked message if the pathway is cleared either by `retryPayload` or `forceResumeReceive`. In the aforementioned instance, we can prevent the proposal from being queued while the bridge is unpaused.

Reply:
If we stores the failed message in `OmnichainGovernanceExecutor` contract insted of LayerZero Endpoint, we may need a function to clear the stored failed message, but as we are implementing `onlyOwner` modifier to the `retryMessage` so we can omit the implementation of `dropFailedMessage`. WDYT?

Reply:
lgtm